specification/app/resource-manager/Microsoft.App/preview/2022-06-01-preview/examples/ContainerApps_Patch.json

```diff
@@ -114,6 +114,133 @@
     }
   },
   "responses": {
+    "200": {
+      "headers": {},
+      "body": {
+        "id": "/subscriptions/34adfa4f-cedf-4dc0-ba29-b6d1a69ab345/resourceGroups/rg/providers/Microsoft.App/containerApps/testcontainerApp0",
+        "name": "testcontainerApp0",
+        "type": "Microsoft.App/containerApps",
+        "location": "East US",
+        "properties": {
+          "provisioningState": "Succeeded",
+          "managedEnvironmentId": "/subscriptions/34adfa4f-cedf-4dc0-ba29-b6d1a69ab345/resourceGroups/rg/providers/Microsoft.App/managedEnvironments/demokube",
```

Comment:
could you also add environmentId because we want to deprecate managedEnvironmentId and use environmentId in the future

Reply:
environmentId change is not in this pr scope